plugins/lsp/language-servers/_mk-lsp.nix

```
@@ -55,6 +55,16 @@ in
         lib.nixvim.mkMaybeUnpackagedOption "plugins.lsp.servers.${name}.package" pkgs name
           package;
 
+      packageFallback = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          When enabled, the language server package will be added to the end of `PATH` (--suffix) instead of the beginning (--prefix).
```

Sorry, I should've spotted this earlier.

Not anything major, but if you're including CLI options in markdown they should be marked as code using `` `backticks` ``, like you've already done with `` `PATH` ``.

```suggestion
          When enabled, the language server package will be added to the end of the `PATH` (`--suffix`) instead of the beginning (`--prefix`).
```

However, in this case we don't really need to quote the CLI options. So it's probably better to do this instead:

```suggestion
          When enabled, the language server package will be added to the end of the `PATH` _(suffix)_ instead of the beginning _(prefix)_.
```